include/swift/Parse/ParsedRawSyntaxNode.h

```diff
@@ -268,16 +251,6 @@ class ParsedRawSyntaxNode {
 
   // Deferred Token Data =====================================================//
 
-  CharSourceRange getDeferredTokenRangeWithTrivia() const {
-    assert(DK == DataKind::DeferredToken);
-    auto leadTriviaLen = DeferredToken.LeadingTrivia.size();
-    auto trailTriviaLen = DeferredToken.TrailingTrivia.size();
-
-    SourceLoc begin = DeferredToken.TokLoc.getAdvancedLoc(-leadTriviaLen);
-    unsigned len = leadTriviaLen + DeferredToken.TokLength + trailTriviaLen;
-
-    return CharSourceRange{begin, len};
-  }
   CharSourceRange getDeferredTokenRange() const {
```

Comment:
As far as I can see, this `getDeferredTokenRange()`(doesn't include trivia) is used only in `ParsedRawSyntaxRecorder` and it's just for calculating the range with trivia after all. Can we abolish this method? If you prefer not to remove this, please add a comment to clarify this does _not_ include trivia ranges.

Reply:
It’ll be gone in the next PR ([here](https://github.com/apple/swift/pull/36277/commits/3a8eed409e9319c0c15fa7adb51144b772773ffc#diff-0b5345cc82c3c5e22980fe5efc348f68015f1005c867b6ae8f1ca52ef6705fecL220)). I know that this splitting between PRs sometimes feels a little bit random. Sorry about that.